charmhelpers/core/hookenv.py

```diff
@@ -50,6 +50,7 @@
 MARKER = object()
 
 cache = {}
+cache_config = None
```

Comment:
This should be moved nearer to the "def config" -- otherwise, it's not really clear what it is.

I'd also name is `_cache_config` to make it clear that it is local.